src/traces/scattergl/attributes.js

```diff
@@ -84,5 +84,17 @@ module.exports = {
     fillcolor: scatterAttrs.fillcolor,
 
     error_y: scatterAttrs.error_y,
-    error_x: scatterAttrs.error_x
+    error_x: scatterAttrs.error_x,
+
+    hoveron: {
```

Comment:
Resolved in 648547f from 

https://github.com/dfcreative/plotly.js/compare/scattergl-lasso...plotly:scattergl-lasso-etienne

@dfcreative you might want to merge that commit in too.